airflow/www/views.py

```diff
@@ -543,7 +543,7 @@ class AirflowBaseView(BaseView):
     }
 
     if not conf.getboolean('core', 'unit_test_mode'):
-        extra_args['sqlite_warning'] = settings.Session.bind.dialect.name == 'sqlite'
+        extra_args['sqlite_warning'] = settings.engine.bind.dialect.name == 'sqlite'
```

Comment:
```suggestion
        extra_args['sqlite_warning'] = settings.engine.dialect.name == 'sqlite'
```

Reply:
The engine is no longer optional. A change made in another PR, so let's see what happens

Reply:
Yeah, I made it non-optional in my models.dag mypy PR.